webapp/components/search_bar.jsx

```diff
@@ -171,7 +171,16 @@ export default class SearchBar extends React.Component {
 
     handleSubmit(e) {
         e.preventDefault();
-        this.handleSearch(this.state.searchTerm.trim());
+        var terms = this.state.searchTerm.trim();
```

Comment:
Can you use `const` here instead of `var`? Locally scopes it and is consistent with the majority of the rest of the codebase